docs/src/architecture/state.md

```diff
@@ -1,10 +1,9 @@
 # State
 
-The state of the network describes the current condition of all individual accounts and note states. It describes what is currently the case.
+The state of the network describes the current condition of all individual accounts and note states. The Miden Node(s) maintain three databases to describe the state:
```

Comment:
I like this 

`It describes what is currently the case.` hehe but maybe it is confusing? You are already the second person who would kick this sentence out. 

Is it confusing or idiomatically not correct?

Reply:
We usually try to write sentences in a more fluent manner such that the user can relate things quickly in their mind. I felt that "is currently the case" was a bit obstructive so I changed it 😅